SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2247,7 +2247,7 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 		}
 		else if (GameInstance->GetSpatialWorkerType() == SpatialConstants::StrategyWorkerType)
 		{
-			StrategySystem->Flush(Connection);
```

Comment:
Nit : Removing the connection from the API might be a bit premature, the strategy worker will need to push updates eventually.

Reply:
I've put the Connection back .